client/landing/stepper/declarative-flow/internals/steps-repository/unified-plans/unified-plans-step.tsx

```diff
@@ -354,10 +360,20 @@ function UnifiedPlansStep( {
 			return headerText;
 		}
 
+		if ( isNewHostedSiteCreationFlow( flowName ) ) {
+			return translate( 'The right plan for the right project' );
```

Comment:
Do we plan to remove these copies from [the original source ](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx#L199)after we merge this?